### ISSUE TITLE
Can't Discard Changes

### ISSUE THREAD

**Issue by Mashimaro7** [desktop/desktop#13661]
### Describe the bug
Write a clear and concise description of what the bug is.

I don't know when this started, but I can no longer discard my changes. It will often show a loading bar, but after it's done it will end, no changes discarded. I've even tried it with 1 change, or 1000 changes. 

### Version & OS

Open the 'About GitHub Desktop' menu to see the GitHub Desktop version. Also include what operating system you are using.

Windows 10 and 

### Steps to reproduce the behavior

1. Right click a change
2. Click discard change

### Expected behavior

Write a clear and concise description of what you expected to happen.

The changes should be reverted and discarded from the list.

### Actual behavior

Write a clear and concise description of what actually happened.

It loads for a minute or two, then nothing happens.

### Screenshots

Add screenshots to help explain your problem, if applicable.

### Logs

Attach your logs by opening the `Help` menu and selecting `Show Logs...`, if applicable.

### Additional context

Basically this issue started about half a month ago. I have to delete the entire project and reclone it all at once. It's pretty annoying, especially because I can't just remove a few changes, I have to restart from a previous commit.

**Comment by steveward** [comment]
Thanks for the report @Mashimaro7! Can you confirm is this is happening in multiple repositories, or only in one specific repository?

Could you also upload the log file from GitHub Desktop so that I could get some more information about this error? To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs`. The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue.

**Comment by Mashimaro7** [comment]
Okay, I had the issue today so here you go.
________________________________

**Comment by steveward** [comment]
@Mashimaro7 unfortunately you will need to upload the log file directly in the issue -- issue attachments are not supported via email replies. Thanks!

**Comment by Mashimaro7** [comment]
Ah, sorry. here you go
[Uploading 2022-01-12.desktop.production.log…]()

**Comment by Mashimaro7** [comment]
Ah, sorry, i don't really know how to upload it lol, i'll upload it to Google drive.

https://drive.google.com/file/d/10tGI0vxt6w7T8_0GFE2VypmkWxA0XgUb/view?usp=sharing

**Comment by steveward** [comment]
Thanks for the log. Can you confirm if this is happening in multiple repositories in GitHub Desktop? 

I'm not seeing any specific error messages in the log file, other than a fetch that's taking a while to complete:

`2022-01-10T16:46:34.592Z - info: [ui] [RepositoryIndicatorUpdater]: Refreshing sidebar indicators for 24 repositories took 104.9s of which 396.9s paused, total 501.9s`

Are any other operations taking longer than average in GitHub Desktop?

There are similar reports in https://github.com/desktop/desktop/issues/7155, but in those cases the discard did complete successfully.

**Comment by Mashimaro7** [comment]
Sorry, I should have updated this. I can't explain it, but it's fixed now. It wasn't working for weeks, I didn't attempt to fix it, it's just... fixed... lol, so... Thanks, i guess? lol, I don't know, i guess it's still something you should look into, but I really don't know what other information to supply... 

Ah, and no, I didn't notice it taking too long on all other operations, I just assumed it took so long because i was discarding 1000+ changes, but discarding wasn't working whether i had 1 change to discard or 1000...

**Comment by steveward** [comment]
Thanks for the update! Since this is resolved I'll close this issue out, but let us know if you run into it again.

**Comment by Jorxx88** [comment]
Having the same issue as Mashimaro. Multiple times this has happened. Using Unreal Engine 5 and not sure what's causing this.

**Comment by Mashimaro7** [comment]
This still happens sometimes, actually. I've just kinda gotten used to ctrl+alt+del ing Github lol

**Comment by Jorxx88** [comment]
Yeah it's really unfortunate it causes all kinds of complications and losses. Horrible for workflow

**Comment by Mashimaro7** [comment]
I know, right? Github Desktop has so many issues, it can be frustrating. Like I've lost an entire day's work just because Github decided to revert my changes to an older version inexplicably lol, and if you click the "merge" button too fast sometimes it leaves the conflicts for you to manually work out on your own :/

**Comment by oguzhantorayev** [comment]
You guys have the Unreal editor open so it's not gonna discard your changes on GitHub. You need to close the editor to do it

**Comment by Mashimaro7** [comment]
No, i'm a Unity dev. Discarding changes works in Unity regardless of whether it's open or not.

________________________________
